### PR TITLE
[zh-cn] sync docs/reference/access-authn-authz/service-accounts-admin.md

### PR DESCRIPTION
**Pull request by RifeWang**
### Issue

Closes: #

### CODE REVIEW

File: content/zh-cn/docs/reference/access-authn-authz/service-accounts-admin.md [hunk 1]
```diff
@@ -134,13 +134,19 @@ stored as extra 'private claims' in the issued JWT.
...
+authentication with that token would fail.
 If the referenced object no longer exists (or its `metadata.uid` does not match),
 the request will not be authenticated.
 -->
 当将令牌绑定到某对象时，该对象的 `metadata.name` 和 `metadata.uid`
 将作为额外的“私有声明”存储在所发布的 JWT 中。
 
 当将被绑定的令牌提供给 kube-apiserver 时，服务帐户身份认证组件将提取并验证这些声明。
+如果所引用的对象或 ServiceAccount 正处于删除中（例如，由于 finalizer 的原因），
+那么在 `.metadata.deletionTimestamp` 时间戳之后的 60 秒（或更长时间）的任何时刻，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/service-accounts-admin.md:148]:
```suggestion
那么在 `.metadata.deletionTimestamp` 时间戳之后的 60 秒（或更长时间）后的某一时刻，
```

**Review by my-git9 — COMMENTED** [review]
/approve

**Pull request merged**